project_templates/stack_package/{{cookiecutter.package_name}}/ups/{{cookiecutter.package_name}}.table

```diff
@@ -0,0 +1,21 @@
+# For each dependency except python, list it here along with its minimum
+# version number.
+# Very common third-party packages (boost, python, swig, doxygen) 
+# and very low-level LSST packages can be assumed to be recursively
+# included by low-level LSST packages such as utils or daf_base.
+# Any other package whose interface is used should be listed explicitly
+# rather than assuming it will be included recursively.
+setupRequired(boost >= 1.47.0)
+setupRequired(utils >= 4.6.0.0)
+setupRequired(pex_exceptions >= 4.6.0.0)
+setupRequired(ndarray)
+
+# The following is boilerplate for all packages.
+# See https://dmtn-001.lsst.io for details on LSST_LIBRARY_PATH.
+{%- if cookiecutter.uses_cpp %}
+# Pure-Python packages only need the last line.
+envPrepend(LD_LIBRARY_PATH, ${PRODUCT_DIR}/lib)
```

Comment:
Hmm.  Above I commented about this, but it looks like you are trying to not print these lines if it's python only, but the python only example definitely has these lines in the table file.